src/ignoreButRetain.js

```diff
@@ -163,6 +161,7 @@ define([
         },
         parseBindElement: function (form, el, path) {
             if (this.data.ignore.active) {
+                path = xpath.parser.parse(path).toXPath();
```

Comment:
I forget, is it possible that this could throw an exception if `path` is not a invalid xpath expression? Would it be worth adding a test for that case?